### PR TITLE
Linkerd Project Graduation Review

### PR DESCRIPTION
**Pull request by leecalcote**

### CODE REVIEW

File: recommendations/linkerd.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+audit](https://github.com/linkerd/linkerd2/blob/main/SECURITY_AUDIT.pdf) in
+June 2019.
+
+## SIG Network recommendation
+
+**Open Questions**
+
+1. Published project scope - https://github.com/linkerd/rfc/issues/37
+1. Roadmap clarification - https://github.com/linkerd/linkerd2/issues/6009
+1. Linkerd mailing list for code of conduct issues? - https://github.com/linkerd/linkerd/wiki/Linkerd-code-of-conduct#moderation
```
Comment (wmorgan) [inline recommendations/linkerd.md:179]: Good catch, I updated this wiki page to point to the private maintainers list, which should be the starting point for moderation discussions.

File: recommendations/linkerd.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+Per the proposal, Linkerd completed a CNCF-sponsored [Cure53 security
+audit](https://github.com/linkerd/linkerd2/blob/main/SECURITY_AUDIT.pdf) in
+June 2019.
+
+## SIG Network recommendation
+
+**Open Questions**
+
+1. Published project scope - https://github.com/linkerd/rfc/issues/37
+1. Roadmap clarification - https://github.com/linkerd/linkerd2/issues/6009
```
Comment (wmorgan) [inline recommendations/linkerd.md:178]: This has been resolved

File: recommendations/linkerd.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+Per the proposal, Linkerd completed a CNCF-sponsored [Cure53 security
+audit](https://github.com/linkerd/linkerd2/blob/main/SECURITY_AUDIT.pdf) in
+June 2019.
+
+## SIG Network recommendation
+
+**Open Questions**
+
+1. Published project scope - https://github.com/linkerd/rfc/issues/37
```
Comment (wmorgan) [inline recommendations/linkerd.md:177]: Have a followup question for you in here. Would love to know more about what you're looking for.

**Comment by wmorgan** [comment]
@leecalcote ping, would love to know if the answers above address your questions! Thank you!

File: recommendations/linkerd.md (hunk 1 above)
Comment (leecalcote) [reply recommendations/linkerd.md:179]: Thank you, sir.

**Comment by wmorgan** [comment]
Thank you, very much appreciated! 🎉

**Pull request merged**